nuget/lib/dependabot/nuget/nuget_config_credential_helpers.rb

```diff
@@ -0,0 +1,70 @@
+# typed: true
+# frozen_string_literal: true
+
+module Dependabot
+  module Nuget
+    module NuGetConfigCredentialHelpers
+      def self.user_nuget_config_path
+        home_directory = Dir.home
+        File.join(home_directory, ".nuget", "NuGet", "NuGet.Config")
```

Comment:
Does the casing works in Linux? Windows is case insensitive so we won't have a problem.

Reply:
This is the exact casing used in Linux and this file is placed during the dotnet install.